storage/innobase/handler/ha_innodb.h

```diff
@@ -188,8 +188,14 @@ class ha_innobase final: public handler
 		key_range*		min_key,
 		key_range*		max_key) override;
 
+	int enable_persistent_count();
+	int disable_persistent_count();
+
+	ha_rows records();
```

Comment:
Are `override` keywords missing here? Did you try to compile with `clang`?

Reply:
I haven't been able to compile with `clang` but the previous version of `ha_innodb.h` that this is based off of omitted the `override` keyword for `records()`; will add though